.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess you accidentally added the `.idea` files?
We should probably just put the entire `.idea` into the root `.gitignore` since it just messes with Android Studio when checking out a branch from somebody else

Reply:
I added `.idea` to `.gitignore` in https://github.com/PhilKes/Notally/commit/e77d347460a14f8fb506336ef477cb7661624923

Reply:
> I guess you accidentally added the `.idea` files? We should probably just put the entire `.idea` into the root `.gitignore` since it just messes with Android Studio when checking out a branch from somebody else

That's strange; it didn’t display in Android Studio when I pushed the code using Git.

Reply:
Yeah happened to me before too, I guess IntelliJ at some point just stages these changes if they are not in the `.gitignore`. 
Perhaps you can just delete then with another commit, and then we can just squash the PR on merge